test-e2e/project-crud.js

```diff
@@ -126,20 +126,42 @@ test('CRUD operations', async (t) => {
     await t.test('getMany', async (st) => {
       const projectId = await manager.createProject()
       const project = await manager.getProject(projectId)
-      const values = new Array(5).fill(null).map(() => {
-        return getUpdateFixture(value)
-      })
-      for (const value of values) {
-        // @ts-ignore
-        await project[schemaName].create(value)
+      const writePromises = []
+      let i = 0
+      while (i++ < CREATE_COUNT) {
```

Comment:
Nit: could we use a traditional `for` loop here?

```suggestion
      for (let i = 0; i < CREATE_COUNT; i++) {
```

Feel free to ignore.

Reply:
No idea why I used while here!